docs/extensions/extensions_api.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Extension API
+title: Extensions API
 category: 62c5cb137dbdad00536291a6
 hidden: true
 ---
 
+## Overview
```

Comment:
The reason for this change is b/c the Readme.com settings for Phylum were recently changed to include an auto-generated Table of Contents along the right hand side of pages. For pages that have more than just a line or two to start with, it seemed wrong to not have a clickable link in the ToC to the "top" of the page. See the current [api](https://docs.phylum.io/docs/api) or [manifest_format](https://docs.phylum.io/docs/manifest_format) pages for that view.

That said, it is just a style choice and not one that is applied consistently. So it can be changed back, if desired.